src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -137,7 +137,7 @@ public boolean isTableBlacklisted(String databaseName, String tableName) {
 	}
 
 	public static boolean isSystemBlacklisted(String databaseName, String tableName) {
-		return "mysql".equals(databaseName) && "ha_health_check".equals(tableName);
+		return "mysql".equals(databaseName) && ("ha_health_check".equals(tableName) || "rds_heartbeat2".equals(tableName));
```

Comment:
perhaps use `startsWith` to match https://github.com/zendesk/maxwell/blob/c59b5e6cf25880716c5d2ba2a164dc6b4ca967d9/src/main/java/com/zendesk/maxwell/replication/MaxwellReplicator.java#L260 ?